Modules/pyexpat.c

```diff
@@ -72,10 +72,11 @@ pyexpat_get_state(PyObject *module)
 
 /* Declarations for objects of type xmlparser */
 
-typedef struct {
+typedef struct xmlparseobject_ {
```

Comment:
```suggestion
typedef struct xmlparseobject {
```

In CPython, we reuse the same name.

Reply:
@picnixz interesting, I did not know that was possible. Upcoming with next push…